index.html

```diff
@@ -429,6 +429,21 @@ <h3 class="card__title">Comments</h3>
               </div>
             </div>
           </label>
+          <label>
+            <input id="experienceUrl" name="experienceUrl" type="radio" value="croupier" class="branded-content-card" />
```

Comment:
If we're not applying specific styles to this element, I'd use a data attribute instead of a class.
Maybe we can also make this more generic by specifying the site URL here. Something like this:
```suggestion
            <input id="experienceUrl" name="experienceUrl" type="radio" value="croupier" data-siteUrl="https://https://playground.mrf.io/templates/branded-content-example.html" />
```